data/campaigns/Heir_To_The_Throne/scenarios/20b_Underground_Channels.cfg

```diff
@@ -268,4 +279,400 @@
+                [/message]
+                [message]
+                    speaker=second_unit
+                    message= _ "What aid might we offer?"
+                [/message]
+                [message]
+                    speaker=unit
+                    message= _ "If you would help rid these caverns of those infernal undead, the Ancient One would be most grateful."
+                [/message]
+                [message]
+                    speaker=second_unit
+                    message= _ "Ancient one?"
+                [/message]
+                [message]
+                    speaker=unit
+                    message= _ "Our elder. Called, by some, Haralamdum."
+                [/message]
+            [/else]
+        [/if]
+
+        {VARIABLE met_wose yes}
+        {VARIABLE silent no}
+        [fire_event]
+            name=objectives
+        [/fire_event]
+
+        [event]
+            name=die
+            [filter]
+                id=Haralamdum
+            [/filter]
+
+            {VARIABLE met_wose no}
+            {VARIABLE silent yes}
+            [fire_event]
+                name=objectives
+            [/fire_event]
+        [/event]
+
+        [event]
+            name=die
+            [filter]
+                id=Muff Argulak
+            [/filter]
+            [filter_second]
+                [not]
+                    side=1
+                [/not]
+            [/filter_second]
+
+            {VARIABLE met_wose no}
+            {VARIABLE silent yes}
+            [fire_event]
+                name=objectives
+            [/fire_event]
+        [/event]
+
+        [event]
+            name=die
+            [filter]
+                id=Muff Argulak
+            [/filter]
+            [filter_second]
+                side=1
+            [/filter_second]
+            [filter_condition]
+                {CHECK_VARIABLE met_wose yes}
+                [have_unit]
+                    id=Haralamdum
+                [/have_unit]
+            [/filter_condition]
+
+            [message]
+                speaker=Haralamdum
+
+                message= _ "Har hum har .. hmm .. well .. har alam dum .. good."
+            [/message]
+
+            [message]
+                speaker=Konrad
+                message= _ "Those undead will bother you no more, Sir Wose."
+            [/message]
+
+            [delay]
+                time=2000
+            [/delay]
+
+            [message]
+                speaker=Delfador
+                message= _ "We still must find the exit from these caverns, Konrad."
+            [/message]
+
+            [message]
+                speaker=Konrad
+                message= _ "Yes, Delfador. I'm afraid we must be going, Sir Wose."
+            [/message]
+
+            [message]
+                speaker=Haralamdum
+                message= _ "I'll be .. hmm dum har .. young Master Delfador? Harum-alam-alam. So it is! Hmm hmm alam .. how is your head?"
```

Comment:
Well, my thought was more that contractions would be seen as "hasty" rather than that woses would speak formally, though that's not to say that I think they wouldn't use more formal phrasing. (I wasn't really thinking about it in general, just noticed this one case.)